### PR TITLE
data:rashmip2408

### PR DESCRIPTION
**Pull request by rashmip2408**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

I really don't know, what to do with this, so I am leaving this as it is.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3811"></a>

### CODE REVIEW

File: data/rashmip2408.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Rashmi Pawar",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Frontend developer **Beginner**. Like to read Books Yoga & Travel",
+  "avatar": "https://github.com/rashmip2408.png"
+  "links": [
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "linkedin.com/in/rashmi-pawar-b4888322b",
```
Comment (Aadarsh805) [inline data/rashmip2408.json:10]: Correct this link to `https://linkedin.com/in/rashmi-pawar-b4888322b`

File: data/rashmip2408.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Rashmi Pawar",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Frontend developer **Beginner**. Like to read Books Yoga & Travel",
+  "avatar": "https://github.com/rashmip2408.png"
```
Comment (Aadarsh805) [inline data/rashmip2408.json:6]: missing comma at the end of line 6

File: data/rashmip2408.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "links": [
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "linkedin.com/in/rashmi-pawar-b4888322b",
+      "icon": "FaLinkedIn"
```
Comment (Aadarsh805) [inline data/rashmip2408.json:11]: fix this LinkedIn icon, use `FaLinkedin`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/rashmip2408.json (hunk 3 above)
Comment (rashmip2408) [reply data/rashmip2408.json:11]: image
commit directly to branch or create new branch again?

Comment (Aadarsh805) [reply data/rashmip2408.json:11]: same branch, yes this `patch-1`

**Review by Aadarsh805 — APPROVED** [review]
Look good  👍

**Comment by rashmip2408** [comment]
> 
Sir but my profile not showing anywhere, how can I do this live!

**Comment by Aadarsh805** [comment]
> > 
> 
> Sir but my profile not showing anywhere, how can I do this live!

It'll be live after it's merged

File: data/rashmip2408.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://linkedin.com/in/rashmi-pawar-b4888322b",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/rashmip2408",
+      "icon": "FaGithub"
+    },
```
Comment (Aadarsh805) [inline data/rashmip2408.json:17]: hey remove this comma at the end of line 17

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**